server/datastore/mysql/migrations/tables/20230926082152_AddWindowsMDMTables.go

```diff
@@ -0,0 +1,69 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20230926082152, Down_20230926082152)
+}
+
+func Up_20230926082152(tx *sql.Tx) error {
+	_, err := tx.Exec(`
+CREATE TABLE windows_mdm_commands (
+    -- command uuid is fleet-generated, so that a command has a unique id as
+    -- soon as it is enqueued. It is unique per device, unlike macOS commands
+    -- that may target multiple hosts. Note that a UUID is 36 chars, but for
+    -- consistency with nano_mdm_commands we use 127 here.
+    command_uuid    VARCHAR(127) NOT NULL,
+    device_id       VARCHAR(255) NOT NULL,
+
+    cmd_verb        SMALLINT UNSIGNED NOT NULL,
+    setting_uri     VARCHAR(255) NOT NULL,
+    setting_value   VARCHAR(255) NOT NULL,
+    priority        SMALLINT UNSIGNED NOT NULL DEFAULT 0,
+
+    created_at      TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    updated_at      TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+
+    PRIMARY KEY (command_uuid)
+)`)
+	if err != nil {
+		return fmt.Errorf("failed to create windows_mdm_commands table: %w", err)
+	}
+
+	_, err = tx.Exec(`
+CREATE TABLE windows_mdm_command_results (
+    command_uuid    VARCHAR(127) NOT NULL,
+    device_id       VARCHAR(255) NOT NULL,
+    session_id      VARCHAR(255) NOT NULL,
+    message_id      VARCHAR(255) NOT NULL,
+    -- this is the MS-MDM protocol-specific command ID, not the Fleet command UUID.
+    command_id      VARCHAR(255) NOT NULL,
+
+    cmd_verb        SMALLINT UNSIGNED NOT NULL,
+    setting_uri     VARCHAR(255) NOT NULL,
+    setting_value   VARCHAR(255) NOT NULL,
+    priority        SMALLINT UNSIGNED NOT NULL DEFAULT 0,
```

Comment:
Any reason we repeat those 4 fields in the command results table vs joining with `windows_mdm_commands` to get them when needed?